tests/custom_hypothesis_support.py

```diff
@@ -63,6 +71,17 @@ def heterogeneous_dictionary(**kwargs):
     return hs.dictionaries(index_values, primitive_values, **kwargs)
 
 
+def non_boolean_operator():
+    """Return a non-boolean operator symbol (string)."""
+    non_bool_operators = ['+', '-', '*', '//', '%', '/', '**', '&', '^', '~', '|', '<<', '>>']
+    return return_one_of(non_bool_operators)
```

Comment:
Better, but instead of using `hs.one_of`, use `hs.sampled_from`. This will eliminate the need for a helper.